From: "Davide G. M. Salvetti" <salve@debian.org>
Date: Sat, 14 Jan 2012 22:57:31 +0100
Subject: preview-image-type: Customize for Debian

* preview.el (preview-image-type): Look for a dvipng executable and use it
  if available.

Rationale: "dvipng is much faster than the combination of Dvips and
Ghostscript", the AUCTeX README states.

Forwarded: not-needed
---
 preview.el.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/preview.el.in b/preview.el.in
index e26f478..3eee565 100644
--- a/preview.el.in
+++ b/preview.el.in
@@ -223,7 +223,7 @@ method, like when PDFTeX was used."
                        (symbol :tag "Emacs image-type")
                        (repeat :inline t :tag "Ghostscript options" string))))
 
-(defcustom preview-image-type 'png
+(defcustom preview-image-type (if (executable-find "dvipng") 'dvipng 'png)
   "Image type to be used in images."
   :group 'preview-gs
   :type (append '(choice)
